### PR TITLE
feat(eigen-client-extra-features): Replace third party dependant tests with mock tests

### PR DESCRIPTION
**Pull request by juan518munoz**
## What ❔

- Ignore existing EigenDA tests inside of library `zksync_da_clients` to prevent a CI failure in case of third party issues (no connection to rpc used in tests)
- Verifier tests now use a `MockVerifierClient` when running on the CI.
- Client tests no longer run on the CI.
- To run previous tests (now ignored) execute:
    ```bash
    cargo test -p zksync_da_clients -- --ignored
    ```

## Why ❔

Prevent CI running longer that it should, or even failing in case a third party service is not available.

## Checklist

- [x] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [x] Code has been formatted via `zkstack dev fmt` and `zkstack dev lint`.

### CODE REVIEW

File: core/node/da_clients/src/eigen/sdk.rs [hunk 1]
```diff
@@ -58,7 +61,22 @@ impl RawEigenClient {
...
+            K256PrivateKey::from_bytes(zksync_types::H256::from_str(
+                &verifier_config.private_key,
+            )?)?,
+            H160::from_str(&verifier_config.svc_manager_addr)?,
+            Verifier::DEFAULT_PRIORITY_FEE_PER_GAS,
+            SLChainId(verifier_config.chain_id),
+            query_client,
+        );
+
+        // let verifier = Verifier::new(verifier_config, MockVerifierClient::new(HashMap::new()))
```
Comment (gianbelinche) [inline core/node/da_clients/src/eigen/sdk.rs:78]:
```suggestion
```

**Review by gianbelinche — APPROVED** [review]

File: core/node/da_clients/src/eigen/client.rs [hunk 2]
```diff
@@ -291,6 +300,7 @@ mod tests {
         assert_eq!(retrieved_data.unwrap(), data);
     }
 
+    #[ignore = "remote dependency"]
```
Comment (juanbono) [inline core/node/da_clients/src/eigen/client.rs:303]: replace it with something more descriptive like:

"Depends on external RPC node"

**Review by juanbono — CHANGES_REQUESTED** [review]
See comments

File: core/node/da_clients/src/eigen/client.rs (hunk 2 above)
Comment (juan518munoz) [reply core/node/da_clients/src/eigen/client.rs]: Done!

**Review by juanbono — APPROVED** [review]

**Pull request merged**